File: Player/AppCompatSampleApp-java/src/main/java/com/brightcove/player/samples/appcompat/java/PlayerFragment.java
package com.brightcove.player.samples.appcompat.java;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import com.brightcove.player.appcompat.BrightcovePlayerFragment;
import com.brightcove.player.model.DeliveryType;
import com.brightcove.player.model.Video;

/**
 * BrightcovePlayerFragment that loads and plays the sample video.
 */
public class PlayerFragment extends BrightcovePlayerFragment {

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View result = inflater.inflate(R.layout.fragment_player, container, false);
        baseVideoView = result.findViewById(R.id.brightcove_video_view);
        super.onCreateView(inflater, container, savedInstanceState);

        Video video = Video.createVideo(getString(R.string.sdk_demo_video_url), DeliveryType.MP4);
        baseVideoView.add(video);
        baseVideoView.getAnalytics().setAccount(getString(R.string.sdk_demo_account));
        baseVideoView.start();

        return result;
    }
}

File: Player/AppCompatSampleApp-kotlin/src/main/java/com/brightcove/player/samples/appcompat/kotlin/PlayerFragment.kt
package com.brightcove.player.samples.appcompat.kotlin

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.brightcove.player.samples.appcompat.kotlin.databinding.FragmentAppCompatBinding
import com.brightcove.player.model.DeliveryType
import com.brightcove.player.model.Video
import com.brightcove.player.view.BrightcovePlayerFragment

/**
 * BrightcovePlayerFragment that loads and plays the sample video.
 */
class PlayerFragment : BrightcovePlayerFragment() {

    private lateinit var binding: FragmentAppCompatBinding

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {

        binding = FragmentAppCompatBinding.inflate(inflater, container, false)
        brightcoveVideoView = binding.brightcoveVideoView

        super.onCreateView(inflater, container, savedInstanceState)
        val video = Video.createVideo(getString(R.string.sdk_demo_video_url), DeliveryType.MP4)
        baseVideoView.add(video)
        baseVideoView.analytics.account = getString(R.string.sdk_demo_account)
        baseVideoView.start()
        return binding.root
    }
}
